csuntechlab/checkpoint
# Check 140 - Wire up create new org form on admin end to call API

# Description
Sign up form now hits `api/register_admin` backend and creates new organization and admin user.

## Type of change

Please delete options that are not relevant.


# How Has This Been Tested?
- Load up the docker environment 
- Visit [http://localhost:8080/#/signup](http://localhost:8080/#/signup)
- Open up a console log
- Fill in form and submit. What should return is a similar payload to the following
```json
{
    "organization": {
        "organization_name": "example@email.com",
        "logo": {}
    },
    "user": {
        "name": "Hulk Hogan",
        "email": "example@email.com"
    }
}
```

# Checklist:
